.github/workflows/test_charts.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           check-latest: true
 
       - name: Set up chart-testing
-        uses: helm/chart-testing-action@v2.7.0
+        uses: helm/chart-testing-action@v2.6.1
```

Comment:
if there is a problem with the version better add a comment explaining that this has to be pinned because ...

Reply:
in the [example](https://github.com/helm/chart-testing-action?tab=readme-ov-file#example-workflow) they use `v2.7.0`, but the actual last version is `2.6.1`.